gradio/interface.py

```diff
@@ -589,8 +582,8 @@ def launch(self, inline=None, inbrowser=None, share=False, debug=False,
                 getpass.getpass("Enter key for encryption: "))
 
         # Store parameters
-        server_name = server_name or self.server_name or networking.LOCALHOST_NAME
```

Comment:
Unclear why this is being done here instead of inside networking. Like with `server_port`

Reply:
Good point